be/src/util/threadpool.cpp

```diff
@@ -481,6 +486,7 @@ Status ThreadPool::update_max_threads(int max_threads) {
 void ThreadPool::dispatch_thread() {
     std::unique_lock l(_lock);
     InsertOrDie(&_threads, Thread::current_thread());
+    _pthread_ids.insert(Thread::current_thread()->pthread_id());
```

Comment:
No need to erase the pthread id from `_pthread_ids` before the thread exit?

Reply:
yes, as pthread_ids will be cleared, no need to be deleted.